modules/terraform/plan_struct.go

```diff
@@ -91,6 +105,18 @@ func parseModulePlannedValues(module *tfjson.StateModule) map[string]*tfjson.Sta
 	return out
 }
 
+// AssertOutputChangesMapKeyExists checks if the given key exists in the map, failing the test if it does not.
+func AssertOutputChangesMapKeyExists(t testing.TestingT, plan *PlanStruct, keyQuery string) {
```

Comment:
`AssertOutputChangesMapKeyExists` - seems to be unused, can be added a test for it?